diff --git a/shared/i18n/index.ts b/shared/i18n/index.ts
index e315ef413..b580ec795 100644
--- a/shared/i18n/index.ts
+++ b/shared/i18n/index.ts
@@ -72,6 +72,10 @@ export const languageOptions: LanguageOption[] = [
     label: "فارسی (Persian)",
     value: "fa_IR",
   },
+  {
+    label: "Русский (Russian)",
+    value: "ru_RU",
+  },
   {
     label: "Svenska (Swedish)",
     value: "sv_SE",
diff --git a/shared/utils/date.ts b/shared/utils/date.ts
index 397b2c7a4..a45d418ae 100644
--- a/shared/utils/date.ts
+++ b/shared/utils/date.ts
@@ -23,6 +23,7 @@ import {
   ptBR,
   pt,
   pl,
+  ru,
   sv,
   tr,
   vi,
@@ -175,6 +176,7 @@ const locales = {
   pt_BR: ptBR,
   pt_PT: pt,
   pl_PL: pl,
+  ru_RU: ru,
   sv_SE: sv,
   tr_TR: tr,
   uk_UA: uk,
